File: client/client_session/client_api/stream_api.py
from .base_api import BaseAPI


class StreamAPI(BaseAPI):

    def request_a_video_package(self):
        self._tcp_channel.write_command("get_video_frame")

    def send_feedback(self, received_packages: int):
        self._tcp_channel.write_command("feedback", received_packages)

File: cliente-java/src/main/java/com/clientejava/clienteSession/clientApi/StreamAPI.java
package com.clientejava.clienteSession.clientApi;

import java.io.IOException;

import com.clientejava.Channel.TCPChannel;

public class StreamAPI extends BaseAPI {

    public StreamAPI(TCPChannel tcpChannel) {
        super(tcpChannel);
    }

    public void requestAVideoPackage() throws IOException {
        tcpChannel.writeCommand("get_video_frame");
    }

    public void sendFeedback(int receivedPackages) throws IOException {
        tcpChannel.writeCommand("feedback", receivedPackages);
    }
}